src/main/java/org/mongojack/JacksonCodecRegistry.java

```diff
@@ -0,0 +1,61 @@
+package org.mongojack;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.mongodb.MongoClient;
+import org.bson.codecs.Codec;
+import org.bson.codecs.DocumentCodec;
+import org.bson.codecs.configuration.CodecRegistries;
+import org.bson.codecs.configuration.CodecRegistry;
+import org.mongojack.internal.MongoJackModule;
+import org.mongojack.internal.stream.JacksonCodec;
+import org.mongojack.internal.stream.JacksonDecoder;
+import org.mongojack.internal.stream.JacksonEncoder;
+
+import javax.swing.text.Document;

```

Comment:
swing?

Reply:
Darn will fix in the morning. Should definitely be Bson.Document

Reply:
Was actually left in from earlier testing... Safe to remove entirely.